src/layout/shell/SidebarButton.tsx

```diff
@@ -39,7 +39,7 @@ export function SidebarButton({
   return (
     <Button
       {...props}
-      onMouseEnter={(event) => {
+      onMouseEnter={(event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
This is not needed, I guess, Typescript should it inherit the types automatically here. If you specifically need it, we are used to importing MouseEvent directly and not the whole React namespace.